src/gui/hud/SecondaryInventory.cpp

```diff
@@ -558,7 +558,8 @@ bool SecondaryInventoryHud::dragEntity(Entity * io, const Vec2s & pos) {
 		}
 	}
 	
-	return false;
+	Set_DragInter(io);
```

Comment:
Is there a reason you only added the `Set_DragInter` call and not the `RemoveFromAllInventories` and `ARX_INVENTORY_IdentifyIO` calls that would have been reached by falling through to `g_playerInventoryHud.dragEntity` before? `RemoveFromAllInventories` might be redundant but at least `ARX_INVENTORY_IdentifyIO` looks like it should be run here.

Sorry for not responding earlier - I didn't notice that you had updated the pr :/